classes/SubscriptionForm.php

```
@@ -0,0 +1,185 @@
+
+    public static function init(): void
+    {
+        if (isset($_GET['mawiblah-resubscribe'])) {
+            self::handleResubscribeConfirmation();
+        }
+    }
+
+    public static function renderForm(array $audienceHashes = [], array $options = []): string
+    {
+        $siteKey        = Settings::recaptchaSiteKey();
+        $recaptcha      = Settings::recaptchaEnabled() && $siteKey;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$recaptcha'. (undefined)

(UnusedLocalVariable)